bugbug/utils.py

```diff
@@ -513,3 +513,14 @@ def escape_markdown(text: str) -> str:
         .replace(")", "\\)")
         .replace("|", "\\|")
     )
+
+
+# Remove business days from date range
+def get_business_days_count(start_date, end_date):
+    np_start_date = np.datetime64(dateutil.parser.parse(start_date), "D")
```

Comment:
Why do we need `dateutil.parser.parse` here?

Reply:
You asked me to remove it from the function call here https://github.com/mozilla/bugbug/pull/3357/files/1afe81039520aad46d508071687ff1cee325b02f#r1148414434

Reply:
Yes, but is it needed at all?

Reply:
I'm not sure what format is being passed to the function call, so I wanted to keep it as close to the original code as I could. Depending on the format of the dates being passed in, it might not be necessary, but I'm not sure where to find an example of what's being passed in.

In my testing, without the parser, the function fails unless the date passed in is in ISO 8601 format

Reply:
Good point. The dates are in ISO 8601 format. Example: https://bugzilla.mozilla.org/rest/bug?id=36539&include_fields=creation_time,history.when

Reply:
Perfect! In that case, I don't believe it's necessary. I'll go ahead and remove it for now!

Reply:
Ah, when I tested the format used here, it gave me a warning: 
`DeprecationWarning: parsing timezone aware datetimes is deprecated; this will raise an error in the future`

Will this be a problem? It was able to work for now, but it looks like in the future, the "Z" after the date passed in may not work.

Reply:
Then it is needed :)  Could you please mention this in a comment?

Reply:
Will do!

Reply:
Would you like me to use the docstrings format, or just a regular comment? I'm unfamiliar with best practices with docstrings

Reply:
in this case just a regular comment

Reply:
Should I use the datetime.parser.parse `ignoretz` parameter? It looks like when you pass in a timezone-aware datetime through it, it keeps it as it passes it to numpy. If I set it to ignore, it won't pass through the timezone to numpy and there's no warning (and no error in the future, should the deprecated feature be removed completely)

Reply:
A work around could be dropping both `np.datetime64()` and `dateutil.parser.parse()`, instead slicing the first 10 characters from the datetime string (i.e., `start_date[:10]`).

Reply:
Aha, smart! All of the times are passed as strings?

Reply:
This runs into a different problem. I'm unable to add 1 to the start and end dates when passing them into `busday_count`, meaning the starting day is included and the ending day is excluded. I believe this is unintended, as when you start on a weekend and end on a weekday, it gives the wrong impression.

For example, I have one test set to start on March 5th, a Sunday, and end on March 10th, a Friday. Using the inclusive start date and exclusive end date, We get a count of 4 (Mon, Tue, Wed, Thu), though if you are told that something will take 4 business days on Sunday, you expect it to be completed on Thursday and not Friday.

I can use `datetime.date.fromisoformat` and `datetime.timedelta` to fix this, but in that case I feel I might as well just use `np.datetime64()` since I already have numpy as a dependency.

Reply:
> or example, I have one test set to start on March 5th, a Sunday, and end on March 10th, a Friday. Using the inclusive start date and exclusive end date, We get a count of 4 (Mon, Tue, Wed, Thu), though if you are told that something will take 4 business days on Sunday, you expect it to be completed on Thursday and not Friday.

I think that is OK. If it is 4 business days, then 4 full business days will end on Thursday and the response will be on Friday ;)